notification-service/src/main/java/org/arpha/notificationservice/configuration/properties/AzureCommunicationProperties.java

```diff
@@ -0,0 +1,18 @@
+package org.arpha.notificationservice.configuration.properties;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+@ConfigurationProperties(prefix = "azure.services.communication")
+@EnableConfigurationProperties(AzureCommunicationProperties.class)
+public class AzureCommunicationProperties {
```

Comment:
Can we change it to records? I think it would look cleaner

Reply:
done